from ..utils.exception import TestRestApiException


class TestDataException(TestRestApiException):
    pass
